superset/models/core.py

```diff
@@ -562,11 +562,11 @@ def alter_positions(dashboard, old_to_new_slc_id_dict):
 
         # override the dashboard
         existing_dashboard = None
-        for dash in session.query(Dashboard).all():
-            if ('remote_id' in dash.params_dict and
-                    dash.params_dict['remote_id'] ==
-                    dashboard_to_import.id):
-                existing_dashboard = dash
+        # for dash in session.query(Dashboard).all():
+        #     if ('remote_id' in dash.params_dict and
+        #             dash.params_dict['remote_id'] ==
+        #             dashboard_to_import.id):
+        #         existing_dashboard = dash
```

Comment:
This code change violates the rule of changing the slug when importing the dashboard. That being said, with same slug user can now import the dashboards. And the side effect to it is a new bug with which user can import dashboards but both dashboards (with different id but same slug) becomes unusable.

Attached is the gif for reference
![import_dashboard_bug](https://user-images.githubusercontent.com/18305698/56505146-65ae3380-6538-11e9-84bc-edc4e74f2285.gif)

Reply:
Nice catch

Reply:
The issue exists without the fix as well. When you import an existing slug for first time it allows you to do the same.

We may need to fix this issue separately.